FND/Places/RealProperty.rdf

```diff
@@ -0,0 +1,226 @@
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-rel-rel;isEvaluatedBy"/>
+				<owl:onClass rdf:resource="&fibo-fnd-plc-rp;RealPropertyAppraisal"/>
+				<owl:minQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">0</owl:minQualifiedCardinality>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label xml:lang="en">real property</rdfs:label>
+		<rdfs:seeAlso rdf:resource="https://www.investopedia.com/terms/r/real-property.asp"/>
+		<skos:definition xml:lang="en">physical asset defined as land together with any structures that are permanently attached to that land, such as houses, trees, fences and other improvements</skos:definition>
+		<cmns-av:adaptedFrom rdf:datatype="&xsd;anyURI">https://www.law.cornell.edu/cfr/text/10/600.101</cmns-av:adaptedFrom>
+		<cmns-av:explanatoryNote xml:lang="en">Real property may be classified according to its general use as residential, commercial, agricultural, industrial, or special purpose. This term is sometimes used synonymously with &apos;real estate&apos;, though not in all circumstances under US law.</cmns-av:explanatoryNote>
+		<cmns-av:explanatoryNote xml:lang="en">Real property typically encompasses both the physical land and everything that lies above, below, or on its surface, including any fixed structures, natural resources, and rights or interests (e.g., mineral rights). There are cases, such as condominiums, in which the interior of the structure is owned by a party that may not own the land. There are also cases in which certain long-term leases have similar characteristics to ownership, but are time-bound.</cmns-av:explanatoryNote>
+		<cmns-av:synonym xml:lang="en">parcel</cmns-av:synonym>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-fnd-plc-rp;RealPropertyAppraisal">
+		<rdfs:subClassOf rdf:resource="&fibo-fnd-arr-asmt;Appraisal"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-dt-oc;hasInput"/>
+				<owl:onClass rdf:resource="&fibo-fnd-plc-rp;PropertyInspection"/>
+				<owl:minQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">0</owl:minQualifiedCardinality>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-rel-rel;evaluates"/>
+				<owl:someValuesFrom rdf:resource="&fibo-fnd-plc-rp;RealProperty"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-rel-rel;isGeneratedBy"/>
+				<owl:someValuesFrom rdf:resource="&fibo-fnd-arr-asmt;Appraiser"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label>real property appraisal</rdfs:label>
+		<skos:definition>value assessment that estimates the amount of money some real property is worth</skos:definition>
+		<cmns-av:explanatoryNote>The valuation uses one or more methodologies and is conducted by an appraiser or technology with a logical model that performs the same function.</cmns-av:explanatoryNote>
```

Comment:
@rivettp  - I think real property appraisals require a human appraiser at the moment. Inspections might be done using technology, such as a drone, and there may be algorithmic analyses to determine valuation and other elements of an appraisal, but if an appraisal is part of a loan, then I think the law requires an actual appraisal / opinion to be signed off by a person so that one has recourse if you want to fight it. Not sure about refinancing - it it is the same institution they may use the original appraisal, though not in Los Altos because the valuations change so much over time. Same for home equity loans - I think they require a human appraisal depending on the dollar amount. I can make it optional, but in the parent ontology for assessments it is required, so I would have to loosen that in both places.